src/store/modules/geometry/helpers.js

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash';
```

Comment:
Consider breaking helpers up into different files so we don't have one giant helpers file